src/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import Heading from '../Heading'
+import NavBars from '../NavBar/NavBar'
```

Comment:
Can you change imports here to just be `import NavBars from '../NavBar'` please? 🙏

Reply:
Fixed. Also fixed all other imports that had doubled links (such as ChartForm/ChartForm, WorkoutForm/WorkoutForm, etc)